HealthcareSystem/PatientWebApp/Validators/SurveyValidator.cs

```diff
@@ -13,24 +13,28 @@ namespace PatientWebApp.Validators
     public class SurveyValidator
     {
         private readonly ISurveyService _surveyService;
-        private readonly DoctorValidator _doctorValidator;
 
         public SurveyValidator(ISurveyService surveyService)
         {
             _surveyService = surveyService;
-            _doctorValidator = new DoctorValidator();
         }
 
         public void ValidateSurveyFields(SurveyDTO surveyDTO)
         {           
             if (EmptyCheck(surveyDTO))
                 throw new ValidationException("Survey cannot be null.");
-            if (!_doctorValidator.IsValidDoctorJmbg(surveyDTO.DoctorJmbg))
-                throw new ValidationException("The doctor does not exist!");
+            if (!IsValidExaminationId(surveyDTO.ExaminationId))
+                throw new ValidationException("The examination does not exist!");
             if (!IsValidateGrades(surveyDTO))
                 throw new ValidationException("The rating is out of range.");
         }
 
+        private bool IsValidExaminationId(int examinationId)
+        {
+            if (examinationId > 0) return true;
+            return false;
+        }
```

Comment:
This check is fairly pointless, wouldn't it make more sense to check that an examination with this id actually exists, and that it's in the correct state for adding a survey (that it's finished and that a survey wasn't already left)?

Reply:
Fixed in 7ae66b5349986db657ee712ed782bf8d87fc9033.